src/ray/common/ray_config_def.h

```diff
@@ -117,7 +117,7 @@ RAY_CONFIG(int64_t, max_direct_call_object_size, 100 * 1024)
 RAY_CONFIG(int64_t, max_grpc_message_size, 100 * 1024 * 1024)
 
 // The min number of retries for direct actor creation tasks. The actual number
```

Comment:
What on earth is this for? Does this mean that we will actually restarted actors even if the user specifies `max_restarts=0`? This doesn't seem like a good idea. Can you `git blame` this and find out why it's here?

Reply:
This was added by Eric Liang I think because we want to treat the failure of an actor creation task a bit differently than the failure of an actor that has started up successfully.

Reply:
There are not many details in the commit message. I'll ask Eric.

Reply:
@ericl this is what I messaged you on slack about.

Reply:
@edoakes from ekl:

> We've always resubmitted the creation task if it failed, similar to normal tasks. I agree it probably makes sense to not do this if num reconstructions is configured to zero, but a lot of applications to flake if we don't reconstruct by default.
> Note that resubmitting just the construction task if it fails is fairly safe; not much can go wrong.
> Reconstructing after the actor has been used, is much dicier.

Reply:
Cool, makes sense to me. Can you please update the comment in the code with this info?